docs/guides/manage-cost.mdx

```diff
@@ -6,55 +6,78 @@ description: How to manage costs of running jobs on QPUs when using the Standard
+<Admonition type="note" title="Notes">
+* This documentation is relevant to the new IBM Quantum&reg; Platform. If you need the previous version, return to the [IBM Quantum Platform Classic documentation.](https://docs.quantum.ibm.com/guides/manage-cost)
+* The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime.  There are no costs associated with the IBM Quantum Platform Open Plan.
 </Admonition>
-</CloudContent>
 
-<LegacyContent>
-The IBM Cloud&reg; Quantum Standard plan is not free, except when running jobs on simulators. Use the information in this topic to help you understand how much you’re paying and how to limit your costs.
+Learn how to limit your costs when using the Standard Plan on IBM Quantum&reg; Platform.
 
-<Admonition type="note">
-  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime on IBM Cloud.  There are no costs associated with the IBM Quantum Platform Open Plan.
+</CloudContent>
+<LegacyContent>
+<Admonition type="note" title="Notes">
+* This documentation is relevant to IBM Quantum&reg; Platform Classic. If you need the newer version, go to the new [IBM Quantum Platform documentation.]( https://quantum.cloud.ibm.com/docs/guides/manage-cost)
+* The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime on IBM Cloud.  There are no costs associated with the IBM Quantum Platform Open Plan.
 </Admonition>
 
-For instructions, see the [Manage costs](https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-cost) topic in the IBM Cloud documentation.
+The IBM Cloud&reg; Quantum Standard plan is not free, except when running jobs on simulators. Use the information in this topic to help you understand how much you’re paying and how to limit your costs.
 
 Learn how to limit your costs when using the Standard Plan on IBM Quantum&reg; Platform.
-
-<Admonition type="note">
-  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime.  There are no costs associated with the IBM Quantum Platform Open Plan.
-</Admonition>
 </LegacyContent>
-<CloudContent>
-Learn how to limit your costs when using the Standard Plan on IBM Quantum&reg; Platform.
 
-<Admonition type="note">
-  The information in this topic only applies to those who are using the Standard Plan for Qiskit Runtime.  There are no costs associated with the IBM Quantum Platform Open Plan.
-</Admonition>
-</CloudContent>
-## Set a cost limit
+<span id="cost-limit"></span>
+## Set or change a cost limit or instance limit
+
+An instance administrator can limit how much is spent. There are several ways to set this limit:
 
-An instance administrator can limit how much is spent.
+* **API**: Set the instance limit in seconds.
+   * New IBM Quantum Platform users: [`PUT /v1/instances/configuration "instance_limit":<seconds>`](/api/qiskit-runtime-rest/tags/instances).
```

Comment:
This link is not working for me.

Reply:
All links to the new QR API REST are not working. Maybe they don't work in the PR preview for some reason?

Reply:
This is an internal issue that my team is working on.